nav2_costmap_2d/include/nav2_costmap_2d/costmap_2d.hpp

```diff
@@ -55,6 +55,16 @@ namespace nav2_costmap_2d
 // convenient for storing x/y point pairs
 struct MapLocation
 {
+  MapLocation()
```

Comment:
I don't see that we need this constructor anywhere. I don't think it's that much of a burden on the user to say `MapLocation foo(0,0)` instead of `MapLocation foo`

Reply:
Agree, will push a change.